Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,863 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    print('Moment-0 ralistic noise spectrum:\t %.3f [a.u]'%moment_0_spec_noise_real )
</pre><p>Same quick-fix typo for "realistic" (^-^)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>